File: 2012/submissions/Wesley Taylor/1/java/problem_1.java
import java.util.Scanner;
import java.util.Arrays;

public class problem_1
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);
        System.out.print("Word to find: ");
        String word_to_find = input.nextLine();

        System.out.print("Sentence: ");
        String sentence = input.nextLine();

        int index = Arrays.asList(sentence.split("\\s+")).indexOf(word_to_find);

        if (index >= 0)
        {
            System.out.println("Word found at index " + index + ".");
        }
        else
        {
            System.out.println("Word not found!");
        }
    }
}

File: 2012/submissions/Wesley Taylor/1/cpp/problem_1.cpp
#include <iostream>
#include <string>
#include <sstream>

using namespace std;

int main(int argc, char *argv[])
{
    string word_to_find;
    cout << "Word to find: ";
    getline(cin, word_to_find);

    cout << "Sentence: ";
    string sentence;
    getline(cin, sentence);
    istringstream words(sentence);

    int index = 0;
    bool found_word = false;
    string current_word;
    while ((words >> current_word) && (current_word != word_to_find))
    {
        ++index;
    }
    if (current_word == word_to_find)
    {
        cout << "Word found at index " << index << ".";
    }
    else
    {
        cout << "Word not found!";
    }
    cout << endl;
    return 0;
}
